hwc2_device/HwcLayer.cpp

```diff
@@ -252,6 +258,35 @@ void HwcLayer::ImportFb() {
     return;
   }
 
+  layer_data_.bi->use_shadow_fds = true;
```

Comment:
for formal patch integration, please replace this hard coding with dynamic mechanism.

Reply:
Yeah, these parts need refinement before integration.